tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -1345,7 +1340,8 @@ def test_consume(self):
                                    timeout=self.msg_timeout)
 
             self.stage_lots_of_failed_consumers()
-            self.stage_hard_restart(producer)
+            RedpandaServiceCloud.concurrent_restart_pods(producer)
```

Comment:
This is not a class method, why is it being called as a class method? Actually it's not clear how this works.